src/graphql/schemas/args/blueprintArgs.ts

```diff
@@ -0,0 +1,26 @@
+import { ArgsType, Field, InputType } from "type-graphql";
+import { withPagination } from "./baseArgs.js";
+import type { OrderOptions } from "../inputs/orderOptions.js";
+import { BasicBlueprintWhereInput } from "../inputs/blueprintInput.js";
+import { Blueprint } from "../typeDefs/blueprintTypeDefs.js";
+import { BlueprintSortOptions } from "../inputs/sortOptions.js";
+
+@InputType()
+export class BlueprintWhereInput extends BasicBlueprintWhereInput {}
+
+@InputType()
+export class BlueprintFetchInput implements OrderOptions<Blueprint> {
+  @Field(() => BlueprintSortOptions, { nullable: true })
+  by?: BlueprintSortOptions;
+}
+
+@ArgsType()
+export class BlueprintArgs {
+  @Field(() => BlueprintWhereInput, { nullable: true })
+  where?: BlueprintWhereInput;
+  // @Field(() => BlueprintFetchInput, { nullable: true })
```

Comment:
Why is this piece commented out?

Reply:
Resolved in https://github.com/hypercerts-org/hypercerts-api/pull/177/commits/194cb7330f09d8396b50450b7bfcb8734b104212